lib/config/folderconfiguration.go

```diff
@@ -233,6 +243,24 @@ func (f *FolderConfiguration) prepare() {
 	if f.MarkerName == "" {
 		f.MarkerName = DefaultMarkerName
 	}
+
+	switch {
+	case f.RawModTimeWindowS > 0:
+		f.cachedModTimeWindow = time.Duration(f.RawModTimeWindowS) * time.Second
+	case runtime.GOOS == "android":
+		usage, err := disk.Usage(f.Filesystem().URI())
+		if err != nil {
+			l.Debugf("Error detecting FS at %v on android, setting mtime window to 2s: %v", f.Path, err)
+			f.cachedModTimeWindow = 2 * time.Second
+			break
+		}
+		if strings.Contains(strings.ToLower(usage.Fstype), "fat") {
+			l.Debugf("Detecting FS at %v on android, found %v, thus setting mtime window to 2s", f.Path, usage.Fstype)
+			f.cachedModTimeWindow = 2 * time.Second
+			break
+		}
+		l.Debugf("Detecting FS at %v on android, found %v, thus leaving mtime window at 0", f.Path, usage.Fstype)
```

Comment:
Hmm... as I reported on the forum, usage.Fstype == "" on Android 8.0 (Moto X Play lux)
Any ideas why that happens @imsodin ? Or does the detection probably need root privileges?
Just tell me if I can help here testing another go code snippet :)

Reply:
I would have expexted that err being nil and fsttpe being "" is impossible. that's something to ask about upstream. for us it means we need to check both error and fstype not being an empty string for now.